packages/server/src/lib/auth.ts

```diff
@@ -0,0 +1,46 @@
+import * as passport from 'passport';
+
+import { Strategy as LocalStrategy } from 'passport-local';
+import { Strategy as JWTStrategy, ExtractJwt } from 'passport-jwt';
+import { UserModel } from '../models';
+
+const dbsecret = process.env.DB_SECRET;
+
+passport.use(
+  new LocalStrategy(
+    { usernameField: 'email', passwordField: 'password' },
+    async (email, password, done) => {
+      try {
+        const user = await UserModel.findOne({ email, password });
```

Comment:
it`s better to split it in two steps. 1 find user by Email, 2. Validate password